builder/amazon/common/access_config.go

```diff
@@ -27,6 +29,7 @@ type AccessConfig struct {
 	SkipValidation       bool   `mapstructure:"skip_region_validation"`
 	SkipMetadataApiCheck bool   `mapstructure:"skip_metadata_api_check"`
 	Token                string `mapstructure:"token"`
+	Insecure             bool   `mapstructure:"insecure"`
```

Comment:
Thank you for point out to that dependencies! Yep, the `amazon-import` was affected by these changes and I'll add that parameter to the doc. But, I don't agree that they affect on `docker` builder and `docker-push` post-processor, because, I think, `cfg` (ecr_login.go) is just a pointer to `common.AccessConfig` and doesn't extend `AwsAccessConfig` with it. And in `amazon-import` the `common.AccessConfig` type is used as `mapstructure:",squash"` which extends the `Config` type. Please, correct me if I'm wrong.